draft-wood-tls-external-psk-importer.md

```diff
@@ -97,20 +100,29 @@ supported HashAlgorithm 'hash', the importer constructs an ImportedIdentity stru
    } ImportedIdentity;
 ~~~
 
+ImportedIdentity.label SHOULD be bound to the protocol for which the key is imported. For example,
+TLS 1.3 over TCP should use "tls13" as the label, whereas QUIC {{!I-D.ietf-quic-transport}} should use
+"quicv1" as the label. Note that this means future versions of TLS will increase the number of
```

Comment:
Should it be "quicv1" or just "tls13"? My recollection is QUIC keeps the "tls13" label for most of it and only changes the label at the end for the traffic secrets, since those are actually a different use of the key. That would suggest "tls13".

In particular, does QUIC pin to TLS 1.3, or can TLS 1.3 rev independently of QUIC? If the former, either probably works. If the latter, I think we'd want it to be whatever TLS 1.3 uses.

Reply:
Your understanding of QUIC is correct! Even with that behavior, I opted with this since I tend to think the original design where "quic" was used as the HKDF prefix was safer. I'll change it here, though, since it does seem to be a minor point.